crates/proof-of-sql/src/utils/mod.rs

```diff
@@ -0,0 +1,3 @@
+//! This module contains utilities for working with the library
+/// Parse DDLs and find bigdecimal columns
+pub mod parse;
```

Comment:
Could this be more descriptive? Perhaps something like ddl_parser? If I were to just look at the file path, I would instinctively think that it should go in the parser crate.